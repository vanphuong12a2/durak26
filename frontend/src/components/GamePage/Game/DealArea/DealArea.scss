.deal-area {
  display: flex;
  align-items: flex-start;

  .card-set {
    margin-right: 5px;
    .card {
      transform: none;
    }
  }
  .card {
    transform: rotate(90deg);
  }
}
