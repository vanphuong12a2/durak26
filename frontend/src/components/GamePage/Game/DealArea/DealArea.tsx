import React from 'react';
import CardSet from '../CardSet/CardSet';

const DealArea = () => {
  return (
    <div className='deal-area'>
      DealArea
      <CardSet cards={[]}/>
    </div>
  );
};

export default DealArea;
