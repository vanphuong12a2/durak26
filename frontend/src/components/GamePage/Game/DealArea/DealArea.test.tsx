import React from 'react';
import {create} from 'react-test-renderer';
import DealArea from './DealArea';

describe('<DealArea />', () => {

  describe('Snapshots', () => {
    it('renders correctly', () => {
      const app = create(<DealArea/>);
      expect(app.toJSON()).toMatchSnapshot();
    });
  });

});
