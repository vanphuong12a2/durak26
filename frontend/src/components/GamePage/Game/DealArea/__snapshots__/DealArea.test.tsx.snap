// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<DealArea /> Snapshots renders correctly 1`] = `
<div
  className="deal-area"
>
  DealArea
  <div>
    Card set
  </div>
</div>
`;
